package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "INTERLIS2 File Highlighter",
     "description": "A VSC extension to support INTERLIS 2",
     "icon": "images/iliIcon.PNG",
-    "publisher": "GeoWerkstatt GmbH",
+    "publisher": "GeoWerkstattGmbH",
```

Comment:
Q: Sieht das dann gut aus im UI? Falls nicht, entweder Underscore für den Abstand benutzen oder das `GmbH` halt weglassen, wenn keine Spaces zugelassen sind :).

Reply:
Falls sie nur im package.json nicht zugelassen sind, würde ich es an den anderen Orten (Manifest, Lizenz) mit Space belassen. Ansonsten lassen wir den Space überall weg, was meinst du?

Reply:
Lizenz garantiert mit Space, Manifest & package.json hätte ich synchron gehalten jedoch ohne GmbH.

Reply:
Bezieht sich `publisher` nicht auf einen Account, der via https://marketplace.visualstudio.com/manage erstellt wird?

Reply:
@tgerstendoerfer Habe ich auch so verstanden. Momentan sehe ich die Extension noch nicht in einem stand welcher im Marketplace veröffentlicht werden sollte. Daher muss `publisher` eh angepasst werden, wenn das geschieht.

Reply:
> @tgerstendoerfer Habe ich auch so verstanden. Momentan sehe ich die Extension noch nicht in einem stand welcher im Marketplace veröffentlicht werden sollte. Daher muss `publisher` eh angepasst werden, wenn das geschieht.

@Philippluca Ich würde vorschlagen, den Account zu registrieren/einzurichten/zu verknüpfen, und wenn es nur ist um den Namen zu reservieren.

Und wissen wir, was es braucht, bis ein veröffentlichbarer Stand erreicht ist?

Reply:
Ich habe bei mir eine Liste was es meiner Meinung nach noch braucht. Dies wird noch in einem Issue dokumentiert.